services/cosi/encode.go

```diff
@@ -0,0 +1,61 @@
+package cosi
+
+import (
+	"bytes"
+	"encoding/base64"
+	"encoding/json"
+	"github.com/dedis/cothority/lib/crypto"
+	"github.com/dedis/cothority/lib/network"
+	"strings"
+)
+
+// MarshalJSON implements golang's JSON marshal interface
+func (s *ServiceResponse) MarshalJSON() ([]byte, error) {
```

Comment:
Do we use JSON or just rely on protobuf for the moment? For me Protobuf would be OK...